exports/contexts/README.md

```diff
@@ -21,3 +21,92 @@ the `bioregistry.export.prefix_maps` module.
 | [`bioregistry.context.jsonld`](bioregistry.context.jsonld) and [`bioregistry.context.ttl`](bioregistry.context.ttl)     | Contains Bioregistry-normalized prefixes (lowercase, punctuation normalized) and the best URI prefixes for each based on the Bioregistry's ruleset. |
 | [`obo.context.jsonld`](obo.context.jsonld) and [`obo.context.ttl`](obo.context.ttl)                                     | Contains prefixes and URI prefixes prioritized for the OBO Foundry and ontology contexts.                                                           |
 | [`obo_synonyms.context.jsonld`](obo_synonyms.context.jsonld) and [`obo_synonyms.context.ttl`](obo_synonyms.context.ttl) | Contains prefixes, prefix's synonyms, and URI prefixes prioritized for the OBO Foundry and ontology contexts.                                       |
+
+## Prescriptive Contexts
+
+As a follow-up to https://github.com/biopragmatics/bioregistry/issues/324, the
+Bioregistry has updated its context export to be more configurable.
+
+### Governance
+
+To ensure stability and reliability for communities that might want to rely on
+these contexts, they were suggested to be only editable by specific people
+responsible for the communities that each context is for. To mitigate the
+possibility of gridlock due to this bottleneck, we institute the following
+rules:
+
+#### Adding a Context
+
+- Anyone is allowed to create a context given they're prepared to follow the
+  governance rules.
+- A discussion for a new context can be made via the Bioregistry's issue
+  tracker, but only a pull request counts as a proposal.
+- New contexts must pass all CI tests before being considered.
+- The Bioregistry Review Team is responsible for reviewing all new context
+  requests and accepts at their discretion.
+- Each context must have a unique key. Similar rules to choosing a good prefix
+  in the Bioregistry apply.
+- Each context must list its maintainers along with their current GitHub, ORCID,
+  and optional email.
+- Each context must include a name and description
+- Contexts should ideally be significantly different from the Bioregistry
```

Comment:
This does not need to be here I think, but doesn't hurt. I think there is a point to be made especially for contexts that are almost the same as bioregistry as this encourages gravitation to using the defaults.

Reply:
Alright good point, if we don't discourage people from doing questionable things, then we can catch the ones who are doing it and try and make this case to them on an _ad-hoc_ basis

Reply:
Removed in ff02f14